examples/lighting-app/mbed/README.md

```diff
@@ -0,0 +1,225 @@
+# CHIP ARM-MbedOS  Lighting Example Application
+
+
+The ARM-MbedOS Lighting Example demonstrates how to remotely control  a white
+dimmable light source. It uses various input primitives to test changing  the lighting and device
+states and LED to show the state of lighting device. Depending on target device also linear  touchpad "slider" is available so light can also be dimmed via PWM output instead of simple ON/OFF toggling. 
```

Comment:
The sentence is convoluted. It can be simplified: 

The example takes advantage of the IO available on board: A buttons press turn the light on or off. Intensity can be displayed and changed depending on the target IO (PWM output, capacitive slider).

Reply:
Done